src/components/scores/index.cjsx

```diff
@@ -201,8 +201,13 @@ Scores = React.createClass
         d.last_name.toLowerCase()
     )
     { headings: scores.data_headings, rows: if sort.asc then sortData else sortData.reverse() }
+
   onColumnResizeEndCallback: (colWidth, columnKey) ->
     @setState({colResizeWidth: colWidth, colResizeKey: columnKey})
+
+  headerType: ->
+    if @props.isConceptCoach then 47 else 92
```

Comment:
Could you add a comment for why the height is different (I'm guessing it is because of the missing heading)?